lib/util/PromisifierUtil.js

```diff
@@ -0,0 +1,66 @@
+import { isFunction, some } from 'min-dash';
+
+export function promisifyIfNecessary(api) {
+
+  return function() {
+
+    // @oguz:
```

Comment:
Isn't the function always the last argument? At least that is common convention in callback style APIs.